pkgs/games/rare/default.nix

```diff
@@ -1,41 +1,30 @@
-{ lib, fetchPypi, buildPythonApplication, makeDesktopItem, copyDesktopItems, qt5
-, pillow, psutil, pypresence, pyqt5, python, qtawesome, requests }:
+{ lib, fetchFromGitHub, buildPythonApplication, qt5
+, psutil, pypresence, pyqt5, python, qtawesome, requests }:
 
 buildPythonApplication rec {
   pname = "rare";
   version = "1.8.9";
 
-  src = fetchPypi {
-    inherit version;
-    pname = "Rare";
-    sha256 = "sha256-UEvGwWjr4FCsvyFz6Db3VnhVS6MS3FYzYSucumzOoEA=";
+  src = fetchFromGitHub {
+    owner = "Dummerle";
+    repo = "Rare";
+    rev = version;
+    sha256 = "sha256-2l8Id+bA5Ugb8+3ioiZ78dUtDusU8cvZEAMhmYBcJFc=";
+    fetchSubmodules = true;
```

Comment:
Is `fetchSubmodules` really necessary?

Reply:
It's one submodule, Legendary. (Which was already bundled in PyPi)
While it's not *required* (since you could copy legendary.src to where Rare looks for it), Legendary's CLI interface is not stable, so it's important that Rare uses a pinned version.